scripts/setup/inline_email_css.py

```diff
@@ -64,14 +75,8 @@ def escape_jinja2_characters(text: str) -> str:
         # template, since we'll end up with 2 copipes of those tags.
         # Thus, we strip this stuff out if the template extends
         # another template.
-        end_block = '</body>\n</html>'
-        start = output.find('{% extends')
-        end = output.rfind(end_block)
-        if start != -1 and end != -1:
-            output = output[start:end]
-        else:
-            # Only the base-default template should not be using this system.
-            assert template == 'email_base_default'
+        if template != 'email_base_default.source.html':
+            output = strip_unnecesary_tags(output)
```

Comment:
On this commit, the tool crashes, I think because of the change in how `template` is encoded here.

```
(zulip-py3-venv) tabbott@coset:~/zulip$ ./scripts/setup/inline_email_css.py 
Traceback (most recent call last):
  File "./scripts/setup/inline_email_css.py", line 79, in <module>
    output = strip_unnecesary_tags(output)
  File "./scripts/setup/inline_email_css.py", line 42, in strip_unnecesary_tags
    raise ValueError("Template does not have %s or %s" % (start_block, end_block))
ValueError: Template does not have {% extends or </body>
</html>
```